SR2021/2020-11-13-sr2021-kickstart-details.md

```diff
@@ -0,0 +1,22 @@
+---
+to: Potential team leaders + signed up teams
+subject: Student Robotics Kickstart Information
+---
+
+Hello there!
+
+We're pleased to announce some more details for the Student Robotics 2020 kickstart.
+
+As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.
+
+## Kickstart
+
+Kickstart will be streamed live on **21st November 2020 at 00:00**. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
```

Comment:
At midnight? Rather you than me :trollface:

Reply:
Placeholder, honest.